### PR TITLE
Feature/reply to comment

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: services/src/test/java/org/exoplatform/task/TestUtils.java [hunk 1]
```diff
@@ -84,6 +83,8 @@ private static void initDB() throws LiquibaseException {
     liquibase.update((String)null);
     liquibase = new Liquibase("db/changelog/task.db.changelog-1.1.0.xml", new ClassLoaderResourceAccessor(), database);
     liquibase.update((String)null);
+    liquibase = new Liquibase("db/changelog/task.db.changelog-1.3.0.xml", new ClassLoaderResourceAccessor(), database);
```
Comment (thomasdelhomenie) [inline services/src/test/java/org/exoplatform/task/TestUtils.java]: Test with task.db.changelog-1.2.1.xml

File: services/src/test/java/org/exoplatform/task/dao/TestCommentDAO.java [hunk 2]
```diff
@@ -78,8 +78,75 @@ public void testCreateComment() {
 
     //Assert.assertEquals(task.getId(), comment.getTask().getId());
   }
+/*
```
Comment (thomasdelhomenie) [inline services/src/test/java/org/exoplatform/task/dao/TestCommentDAO.java]: To fix :)

File: task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less [hunk 3]
```diff
@@ -819,6 +819,12 @@ li[data-hiddenlabel="false"] {
 //comment section
 .commentList {
 
+  .SubCommentBlock {
```
Comment (thomasdelhomenie) [inline task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: No upper character at the beginning.

File: services/src/test/java/org/exoplatform/task/TestUtils.java (hunk 1 above)
Comment (boubaker) [reply services/src/test/java/org/exoplatform/task/TestUtils.java]: To be fixed apart IMO.
The exception with this is:
liquibase.exception.DatabaseException: Table "SOC_APPS" not found; SQL statement:
UPDATE SOC_APPS SET REMOVABLE = true where APP_ID = 'TaskManagementApplication' [42102-145] [Failed SQL: UPDATE SOC_APPS SET REMOVABLE = true where APP_ID = 'TaskManagementApplication']
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:301)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:55)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:107)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1251)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1234)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:554)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:51)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:73)
	at liquibase.Liquibase.update(Liquibase.java:212)
	at liquibase.Liquibase.update(Liquibase.java:192)
	at liquibase.Liquibase.update(Liquibase.java:188)
	at liquibase.Liquibase.update(Liquibase.java:181)
	at org.exoplatform.task.TestUtils.initDB(TestUtils.java:87)
	at org.exoplatform.task.TestUtils.initH2DB(TestUtils.java:66)
	at org.exoplatform.task.AbstractTest.createTable(AbstractTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: org.h2.jdbc.JdbcSQLException: Table "SOC_APPS" not found; SQL statement:
UPDATE SOC_APPS SET REMOVABLE = true where APP_ID = 'TaskManagementApplication' [42102-145]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.Parser.readTableOrView(Parser.java:4514)
	at org.h2.command.Parser.readTableOrView(Parser.java:4492)
	at org.h2.command.Parser.readSimpleTableFilter(Parser.java:684)
	at org.h2.command.Parser.parseUpdate(Parser.java:642)
	at org.h2.command.Parser.parsePrepared(Parser.java:413)
	at org.h2.command.Parser.parse(Parser.java:274)
	at org.h2.command.Parser.parse(Parser.java:246)
	at org.h2.command.Parser.prepare(Parser.java:200)
	at org.h2.command.Parser.prepareCommand(Parser.java:213)
	at org.h2.engine.Session.prepareLocal(Session.java:423)
	at org.h2.engine.Session.prepareCommand(Session.java:373)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1056)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:165)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:153)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:299)
	... 31 more

File: task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less (hunk 3 above)
Comment (boubaker) [reply task-management/src/main/java/org/exoplatform/task/management/assets/less/style.less]: done

File: services/src/test/java/org/exoplatform/task/dao/TestCommentDAO.java (hunk 2 above)
Comment (boubaker) [reply services/src/test/java/org/exoplatform/task/dao/TestCommentDAO.java]: done

File: services/src/test/java/org/exoplatform/task/TestUtils.java (hunk 1 above)
Comment (thomasdelhomenie) [reply services/src/test/java/org/exoplatform/task/TestUtils.java]: ok, please open a dedicated issue.

**Pull request merged**